scripts/image

```diff
@@ -95,13 +95,21 @@ if [ -n "${CUSTOM_VERSION}" ]; then
 fi
 
 LIBREELEC_ARCH="${DEVICE:-${PROJECT}}.${TARGET_ARCH}"
-TARGET_VERSION="${LIBREELEC_ARCH}-${LIBREELEC_VERSION}"
+if [ "${LIBREELEC_BUILD}" = "official" ]; then
```

Comment:
I would change this test to be:
```suggestion
if [ "${OFFICIAL}" = "yes" ]; then
```
as that avoids encoding `"official"` more than once.